### ISSUE TITLE
Clipboard issues with compression, compression throws an error in the log.

### ISSUE THREAD

**Issue by nentibusarchitectura** [Xpra-org/xpra#3824]
**Describe the bug**
When I select a large amount of text, for example in a remote terminal in the server and try to copy the text to a local editor in the client machine nothing happens, no text is pasted.

I can see these errors in the server log:

Server error log:
```
Apr 17 17:31:45 krang xpra[802]: Error during encoding:
Apr 17 17:31:45 krang xpra[802]: Traceback (most recent call last):
Apr 17 17:31:45 krang xpra[802]:   File "/usr/lib/python3/dist-packages/xpra/server/source/client_connection.py", line 300, in encode_loop
Apr 17 17:31:45 krang xpra[802]:   File "/usr/lib/python3/dist-packages/xpra/server/source/clipboard_connection.py", line 134, in compress_clipboard
Apr 17 17:31:45 krang xpra[802]:     packet[i] = compressed_wrapper(item.datatype, item.data, level=9, can_inline=False, brotli=True)
Apr 17 17:31:45 krang xpra[802]:   File "/usr/lib/python3/dist-packages/xpra/net/compression.py", line 189, in compressed_wrapper
Apr 17 17:31:45 krang xpra[802]:     cl, cdata = c.compress(data, level)
Apr 17 17:31:45 krang xpra[802]:   File "/usr/lib/python3/dist-packages/xpra/net/compression.py", line 53, in brotli_compress_shim
Apr 17 17:31:45 krang xpra[802]:     return level | BROTLI_FLAG, brotli_compress(packet, quality=level)
Apr 17 17:31:45 krang xpra[802]:   File "xpra/net/brotli/compressor.pyx", line 106, in xpra.net.brotli.compressor.compress
Apr 17 17:31:45 krang xpra[802]: ValueError: brotli compression failed: 0
Apr 17 17:31:47 krang xpra[802]: client  26 @57.943 Warning: remote clipboard request timed out
Apr 17 17:31:47 krang xpra[802]: client  26 @57.943  request id 12, selection=CLIPBOARD, target=STRING
```
Client messages:
```
2023-04-17 17:36:30,898 Warning: remote clipboard request timed out
2023-04-17 17:36:30,898  request id 1, selection=CLIPBOARD, target=UTF8_STRING
```
**To Reproduce**
Select large amount of text from remote terminal, try to paste on the client, nothing happens.

**System Information (please complete the following information):**
 - Server OS: Ubuntu 20.04.6 LTS
 - Client OS: Ubuntu 20.04.6 LTS
 - Xpra Server Version: 4.4.4-r0-1
 - Xpra Client Version: 4.4.4-r0-1

**Comment by nentibusarchitectura** [comment]
Adding:

--env=XPRA_MIN_CLIPBOARD_COMPRESS_SIZE=99999999"

To the xpra client invocation does the trick of not triggering the compression so I can work with xpra again.

**Comment by totaam** [comment]
I believe that this is a bug with the brotli bindings in Ubuntu 20.04, other distros are not affected.

**Comment by Rush-iam** [comment]
+ Ubuntu 22.04

The log is silent if brotli is not installed in the system and prints that error if brotli is installed

**Comment by colincross** [comment]
Will the fix go to the v4.4.x branch too?

**Comment by totaam** [comment]
> Will the fix go to the v4.4.x branch too?

Yes, assuming all the builds from git master go well and pass the tests.

**Comment by totaam** [comment]
4.4.x: cb1819ee5c0feb447276ed8c09c0b0ee99479fb7
